### PR TITLE
feature/fcsc-notice-endpoints

### PR DESCRIPTION
**Pull request by Akalanka47000**
Added endpoints to manage fcsc-web notices

Get Notices - {domain}/notices/ - GET
Add Notice - {domain}/notices/ - POST
    - Payload : {
                          "title":"Notice "Title",
                          "body":"Blah Blah Blah",
                          "category": ["Latest","Older"] (Enum)
                       }
Edit Notice - {domain}/notices/id - PUT
Delete Notice - {domain}/notices/id - DELETE

Add, edit and delete notices can only be used by admins, or publishers whose club is FCSC

### CODE REVIEW

File: src/api/fcsc-notices/notice.service.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+/**
+ * Create notice in db
+ * @param req
+ * @returns {Promise<Document<any>>}
+ */
+
+const addNotice = async (req) => {
+  if (req.user.role != 'Admin' && req.user.faculty != 'FCSC') {
+    throw { message: 'You are not authorized to access this endpoint' };
+  }
```
Comment (sithumonline) [inline src/api/fcsc-notices/notice.service.js:12]: Can you DRY this?

Comment (Akalanka47000) [reply src/api/fcsc-notices/notice.service.js:12]: put these error messages in a util file?

Comment (sithumonline) [reply src/api/fcsc-notices/notice.service.js:12]: Create separate func, yha like mv to unit

Comment (Akalanka47000) [reply src/api/fcsc-notices/notice.service.js]: check now

Comment (sithumonline) [reply src/api/fcsc-notices/notice.service.js]: Can we do this refactor to the other handlers?

Comment (Akalanka47000) [reply src/api/fcsc-notices/notice.service.js]: moved all validations to a validator util file

**Review by sithumonline — APPROVED** [review]
LGTM

**Pull request merged**